README.md

```diff
@@ -1 +1,46 @@
-# tf_codebuild
\ No newline at end of file
+# tf_codebuild
+
+Terraform config to create codebuild project for codepipeline
+
+## Usage
+
+Include this repository as a module in your existing terraform code:
+
+```
+module "build" {
+  source    = "git::https://github.com/cloudposse/tf_codebuild.git"
+  namespace = "general"
+  name      = "ci"
+  stage     = "staging"
+
+  image         = "apline"
+  instance_size = "BUILD_GENERAL1_SMALL"
+}
+```
+
+Grant propriate permsissions to s3
+
+```
+resource "aws_iam_role_policy_attachment" "codebuild_s3" {
+  role   = "${module.build.role_arn}"
+  policy_arn = "${aws_iam_policy.s3.arn}"
+}
+```
+
+## Input
+
+|      Name     |        Default       |                                                    Decription                                                    |
```

Comment:
No because justify depends of this part

Justify central
```
|Name|
|:-------------:| 
|Value|
```
Justify left

```
|Name|
|-------------| 
|Value|
```